### ISSUE TITLE
MaxEnt in Biomod2 - Model runs (NbRunEval)

### ISSUE THREAD

**Issue by luispedrosantiago** [biomodhub/biomod2#366]
Hi all,

I have one question about Biomod2 MaxEnt replicates (NbRunEval argument of BIOMOD_Modeling()). Do they involve different train/test splits or are these models with different covariates?

I ran some SDMs and a reviewer commented that if I only have different train/test splits I might actually end up with a more biased model.

Thanks in advance!

Luis

**Comment by MayaGueguen** [comment]
Hello Luis,

Sorry for the late answer...
If you still have questions about this topic, I'm not sure I understand your question :eyes: 
Could you be a bit more explicit ?
Also, you are refering to `NbRunEval` which is an old parameter of the [BIOMOD_Modeling](https://biomodhub.github.io/biomod2/reference/BIOMOD_Modeling.html) function now replaced by `CV.nb.rep`. Have you updated `biomod2` lately ?
Version 4.2-4 is available on CRAN and 4.2-5 on github.

Maya

**Comment by luispedrosantiago** [comment]
Hi Maya,

I realise now I didn't phrase it very well, so let me rephrase my question.
(This concerns the older version of biomod2, as the new one was producing loads of errors when I tried to change everything to use the new functions. As I was already in the middle of my PhD and very limited in terms of my timetable, I couldn't go after the reason why those errors and bugs were happening).

Say that I have this piece of code: ensemble_modelscores <- get_evaluations(BIOMOD_EnsembleModeling()). 
The output I get is a little table with results for TSS and ROC, threshold, sensitivity and specificity:
![Capture](https://github.com/biomodhub/biomod2/assets/74597149/bc1437b8-8ddc-42d0-af4a-c3a3d25c6904)

This regards the test dataset. What should I do to find the same sort of results for the training data?

Thanks!

Luis

**Comment by MayaGueguen** [comment]
Thank you for rephrasing :pray: 

Unfortunately, in the older version, evaluation values for calibration data were not returned...
So you won't be able to find them within the files. The only option for you is to calculate them yourself :grimacing: 

We improved that in the new versions.
Do not hesitate to try and test it some day when you have more time :eyes: 

Sorry for the inconvenience..
Maya

**Comment by luispedrosantiago** [comment]
Thank you Maya!

May I suggest something?
I am aware that you have a tutorial concerning this new version of biomod2, but I wonder if you would consider making a video demonstrating the main functionalities of this new version. Just like the one that can be found on youtube (the thing is that that youtube video concerns the old version). I am particularly drawn to videos, as their visualisation allows me to then expand on the use of other functionalities of the packages I use, and I am sure there are more people like me who would benefit from that.

Just a suggestion!

All the best,
Luis

Luís Santiago

PhD candidate in Biodiversity Management | Durrell Institute of Conservation and Ecology (DICE)
School of Anthropology and Conservation, University of Kent
Kent, CT2 7NR

Amazonian Mammal Research Group | National Institute of Amazonian Research
Manaus
***@***.***

**Comment by MayaGueguen** [comment]
This a very good idea indeed, and it is actually on my to-do list :smiley: 
Good to know that it will be valuable for some !
I plan to do that in the coming months, before summer.
Of course I'll keep you updated :envelope:

**Comment by luispedrosantiago** [comment]
Thank you so much! :D

Luís Santiago

PhD candidate in Biodiversity Management | Durrell Institute of Conservation and Ecology (DICE)
School of Anthropology and Conservation, University of Kent
Kent, CT2 7NR

Amazonian Mammal Research Group | National Institute of Amazonian Research
Manaus
***@***.***
